java/demo/src/com/google/phonenumbers/PhoneNumberParserServlet.java

```diff
@@ -200,7 +200,7 @@ private StringBuilder getOutputForSingleNumber(
       } else {
         appendLine("Result from isValidNumber()", Boolean.toString(isNumberValid), output);
         if (isNumberValid) {
-          if (!defaultCountry.isEmpty() && defaultCountry != "ZZ") {
```

Comment:
This is the demo, which requires appengine, and I don't think you can run that without using java 7 anyway?